backend/api/Controllers/RobotController.cs

```diff
@@ -180,7 +184,7 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] Robot robot)
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public async Task<ActionResult<Report>> StartMission(
```

Comment:
Seems strange that we ask for a report here and not a mission object.
Not something to refactor now, but I am commenting so I don't forget it during discussion.

Reply:
I tend to agree with this, the report can then be looked up based on the mission ID if needed